library/objective-c/EnvoyHTTPStreamImpl.m

```diff
@@ -143,9 +150,9 @@ - (instancetype)initWithHandle:(envoy_stream_t)handle
 
   // Create native callbacks
   // TODO(goaway) fix this up to call ios_on_send_window_available
-  envoy_http_callbacks native_callbacks = {ios_on_headers,  ios_on_data,   ios_on_metadata,
-                                           ios_on_trailers, ios_on_error,  ios_on_complete,
-                                           ios_on_cancel,   ios_on_cancel, context};
```

Comment:
This was a (awful) hack, because we needed a function with a valid signature. It would be better to use an explicit no-op callback for on_send_window_available until we have time to wire it through.

Reply:
Will do.

Reply:
Done in d5867ef